json/2.4/modules/ROOT/pages/json-schema-validation.adoc

```diff
@@ -3,7 +3,7 @@
 The JSON module *Validate schema* operation (`<json:validate-schema>`) validates that the input content is compliant with a given JSON schema.
 
 [NOTE]
-Supported schema versions are: Draft 3, Draft 4, Draft 6, Draft 7, Draft 2019-09 and Draft 2020-12. +
+Supported schema versions are: Draft 3, Draft 4, Draft 6, Draft 7, Draft 2019-09 and Draft 2020-12. Draft 6, Draft 7, Draft 2019-09, and Draft 2020-12 have a different payload response than Draft 3 and Draft 4.
```

Comment:
Is it necessary to list all of the drafts that are different from Draft 3 and Draft 4? 
For example, could you instead simplify it and say:
Drafts 3 and 4 have a different payload response than the other drafts.